### PR TITLE
Exc02 03

### PR DESCRIPTION
**Pull request by majkel84**

### CODE REVIEW

File: shm/src/player.cpp [hunk 1]
```diff
@@ -14,10 +14,11 @@ void Player::addMoney(const size_t& money) {
...
 size_t Player::countSpace() {
-    size_t totalAmount = std::accumulate(ship_->getCargo().begin(), ship_->getCargo().end(), 0,
+    /*size_t totalAmount = std::accumulate(ship_->getCargo().begin(), ship_->getCargo().end(), 0,
                                          [](const Cargo& item1, const Cargo& item2) {
                                              return item1.getAmount() + item2.getAmount();
                                          });
 
-    return ship_->getCapacity() - totalAmount;
+    return ship_->getCapacity() - totalAmount;*/
+    return 10;
```
Comment (mariomoore) [inline shm/src/player.cpp:27]: Easy solution:
```
size_t totalAmount = 0;
for (auto it = ship_->getCargo().begin(); it != ship_->getCargo().end(); ++it) {
    totalAmount += (*it)->getAmount();
}
return ship_->getCapacity() - totalAmount;
```

File: shm/src/main.cpp [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+    Fruit fruit("apple", 10, 2.3, 30);
+    std::cout << fruit.getTimeToRotten();
+    --fruit;
+    std::cout << fruit.getTimeToRotten();
+    fruit.nextDay();
+    std::cout << fruit.getTimeToRotten();
+    auto fruitPtr = std::make_shared<Fruit>(fruit);
+    Ship ship(01, "sheeeep", 40, 50, 50);
+    ship.load(fruitPtr);
+    std::cout << fruit;
```
Comment (mariomoore) [inline shm/src/main.cpp]: You can't print fruit ;)

Comment (majkel84) [reply shm/src/main.cpp]: i think i can, but for now i don't know how :)
main is just for testing purpose DO NOT MERGE IT

File: shm/src/player.cpp (hunk 1 above)
Comment (majkel84) [reply shm/src/player.cpp:27]: Player is work of Jakub, it's here by mistake, DONT MERGE PLAYER FROM THIS BRANCH

**Pull request merged**